apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/dao/state/smc/SmcContractEventLogTable.java

```diff
@@ -219,7 +219,7 @@ public SmcContractEventLogEntry getEntry(long logId) {
             sql.append(" AND e.name = ? ");
         }
 
-        if (blockRange.to().intValue() > 0) {
+        if (blockRange.to() > 0) {
```

Comment:
I meant to change blockRange.to() > 0 into 'blockRange.isUpperBoundSet()' would be more correct for incapsulation of the PositiveRange object

Reply:
added isTopBoundarySet() and isBottomBoundarySet() methods